docs: update macos uninstall (#49238)
gravitational/teleport: docs/pages/admin-guides/management/admin/uninstall-teleport.mdx

  You can use `dirname $(which teleport)` to look this up automatically.
  </Admonition>

>   Remove the Teleport binaries and links to Teleport software from the machine:

Author: how's this @r0mant  or do we need more specifics?

Author: @r0mant when you get a chacne can you take a look?

  ```

  <Admonition type="notice" title="Uninstall macOS client tools">

>   You may have Teleport in Applications folder if you:

Reviewer: ```suggestion
  You may have Teleport software in the `/Applications` folder if you:
```

>   - Installed from a macOS tar ball for v17+ that includes `tsh.app` and `tctl.app`

Reviewer: ```suggestion
  - Installed from a macOS tarball for v17+ that includes `tsh.app` and `tctl.app`
```

>   - Installed  macOS `tsh` client-only package for v16 or older versions.

Reviewer: ```suggestion
  - Installed the macOS `tsh` client-only package for v16 or older versions.
```

  - Installed Teleport Connect for macOS  
     
  You can remove those with these commands:

  ```code
  $ sudo rm -rf /Applications/tsh.app

>   $ sudo rm -rf /Applications/tctl.app

Reviewer 2: Should we include commands to remove tsh/tctl symlinks from /usr/local/bin too?
